fabfile/setup.py

```diff
@@ -87,6 +88,19 @@ def move_vagrantfile_to_project_dir():
 
 
 @task
+def create_pem_file():
+    env.settings = get_fab_settings()
+    pem = generate_ssh_keypair(pem_only=True)
+    project_name = env.settings.get('project_name')
+    with open("./{}.pem".format(project_name), 'w') as key:
+        key.write(pem)
+        os.chmod("./{}.pem".format(project_name), 0400)
+
+    print("".join(("PEM-file './{}.pem' added! Add this".format(project_name),
```

Comment:
might want to add a github issue for automatically adding the key to the server - basically the user would have to input the root password of the server while this is happening, but it'd be a one-time only thing during setup and would take a way an annoying part of manual work

a second issue, might be that we want to move over to be purely ssh key based auth and actually remove the root user's password altogether - makes it more concerning that we don't loose this pem file, but it's a lot more secure